portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase

```diff
@@ -1096,64 +1684,311 @@ definition {
 				locator1 = "DataSet#FIELDS_TABLE");
 		}
 
-		task ("When the user Delete Headless Item Action item") {
-			DataSetAdmin.deleteActionEntry(actionName = "Headless Item Action");
+		task ("When the user Delete Headless Item Action item") {
+			DataSetAdmin.deleteActionEntry(actionName = "Headless Item Action");
+		}
+
+		task ("Then the list of action items is empty") {
+			AssertElementNotPresent(
+				key_position = 1,
+				keyName = "Headless Item Action",
+				locator1 = "DataSet#FIELDS_TABLE");
+		}
+	}
+
+	@description = "LPS-192956 Confirm that when the user clicks on the Delete option of the Modal item action, the item action is deleted"
```

Comment:
This test is not needed. Put this test scenario as Low in Jira ticket and remove it.